unit-tests/live/rec-play/test-record-software-device.py

```diff
@@ -0,0 +1,143 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Can we maybe split the logic into functions and add some comments.

It will help understand the flow..

prepare a depth image..
prepare a motion image

record frames
play frames
compare_frames (original, recorded)

something like that.. will look much better IMO